### PR TITLE
Various fixes on navbar. Outlined on the navbar issue on sprint 11

### PR DESCRIPTION
**Pull request by StewartMckenzie**

### CODE REVIEW

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -37,16 +42,16 @@ function Navbar() {
       <nav className='navbar'>
         <div className='spaceHolder'></div>
         <div className='navbar-container'>
-          <Link to='/' className={click ? 'mobile-logo active' : 'navbar-logo'}>
+          <Link to='/' className={setClassName('mobile-logo active','navbar-logo') }>
```
Comment (DandyReyes) [inline src/components/Navbar/Navbar.js:45]: in your setClassName parameter function, the first parameter is **normal** and the second is **active** but when I see this function passing the parameters, isn't the parameters backwards?

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -100,7 +105,7 @@ function Navbar() {
               </Link>
             </li>
 
-            <li className={click ? 'nav-item' : 'hidden'}>
+            <li className={setClassName('nav-item','hidden') }>
```
Comment (DandyReyes) [inline src/components/Navbar/Navbar.js:108]: This doesn't have **active** anywhere in the parameter, maybe making another function name with a more meaningful title will work? Or you could work the parameters in the current function to have more meaningful and general names.

**Review by DandyReyes — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (StewartMckenzie) [reply src/components/Navbar/Navbar.js]: Updated in the latest commit!

File: src/components/Navbar/Navbar.js (hunk 2 above)
Comment (StewartMckenzie) [reply src/components/Navbar/Navbar.js]: Updated in the latest commit!

**Review by DandyReyes — APPROVED** [review]

**Pull request merged**